nx584/config.ini

```diff
@@ -13,15 +13,14 @@ euro_date_format = True
 # (This is useful with the NX590 as it disconnects idle connections in a
 # short period of time)
 # Defaults to 120 seconds
-# idle_time_heartbeat_seconds = 20
+idle_time_heartbeat_seconds = 120
 
 #[email]
 #fromaddr = security@foo.com
 #smtphost = imap.foo.com
 
 [zones]
-# Zone names
-1 = VZ/Door
-2 = VZ/Others
-3 = VZ/Stairs
-4 = VZ/Bedroom
+1 = NX Main entrance
```

Comment:
Ideally we should just get rid of the need to have a `config.ini` in the first place. If you make the `[zones]` stuff somehow a UI parameter (with `bashio`) then we should be able to assemble a `config.ini` instance in `run.sh` and pass it to the NX server.

But this can wait for a follow up release.

Reply:
I wanted to put whole file into config tab, but it was not easy, so I change the config to map a folder and store there the config.ini file.

Ideally I would like to put all configurable parameters in yaml format (config tab) and dynamically generate the config.ini file with its contents.